aperturedb/test_postgres.go

```diff
@@ -0,0 +1,13 @@
+//go:build test_db_postgres
+// +build test_db_postgres
```

Comment:
Should we also run these unit tests in the CI?

Reply:
totally, added to CI